src/test/kotlin/step3/racingcar/CarTest.kt

```diff
@@ -0,0 +1,29 @@
+package step3.racingcar
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class CarTest {
+
+    @Test
+    fun `자동차 이동 실패 테스트`() {
+        val randMoveCount = (0..9).random()
+        val car = Car(FailEngine())
+        repeat(randMoveCount) {
+            car.move()
+        }
+
+        assertEquals(car.movements.count { !it }, randMoveCount)
```

Comment:
```suggestion
        val car = Car(FailEngine())

        car.move()

        assertThat(car.movements).containsOnlyOnce(false)
```

자동차 이동 실패에 대한 테스트인 경우는 이렇게도 표현이 가능할 것 같아요! 😊